.eslintrc

```diff
@@ -1,49 +1,40 @@
 {
   "extends": "airbnb",
```

Comment:
I actually was doing the exact same issue :D
But I came to the conclusion that `"extends": "eslint:recommended"` would get rid of some dependencies introduced by `airbnb` which we likely won't need anyway. Opinions?